pogom/search.py

```
@@ -1293,15 +1330,14 @@ def get_api_version(args):
         s = requests.Session()
         s.mount('https://',
                 HTTPAdapter(max_retries=Retry(total=3,
-                                              backoff_factor=0.1,
+                                              backoff_factor=0.4,
                                               status_forcelist=[500, 502,
                                                                 503, 504])))
         r = s.get(
             'https://pgorelease.nianticlabs.com/plfe/version',
             proxies=proxies,
             verify=False)
-        return r.text[2:] if (r.status_code == requests.codes.ok and
-                              r.text[2:].count('.') == 2) else 0
+        return r.text[2:] if r.status_code == requests.codes.ok else None
     except Exception as e:
         log.warning('error on API check: %s', repr(e))
-        return 0
+        return None
```

Instead of returning `None` on failure, we can return `False`. The rest of the logic would stay the same, but I find `False` to be clearer. Thoughts?